pupil_src/launchables/player.py

```diff
@@ -619,8 +622,12 @@ def on_drop(window, count, paths):
             glfw.glfwSwapBuffers(window)
 
             if rec_dir:
-                update_recording_to_recent(rec_dir)
-                glfw.glfwSetWindowShouldClose(window, True)
+                try:
+                    update_recording_to_recent(rec_dir)
+                except AssertionError as err:
```

Comment:
raising the error to log is good. Also keeping the window open and displaying the error in the text field could be better no?